Pidaq/UI/Stability/stability_dash/callbacks/control_callbacks.py

```diff
@@ -0,0 +1,138 @@
+import json
+import os
+
+import dash
+from dash.dependencies import Input, Output, State
+from dash.dash import no_update 
+
+from stability_dash.app import app
+from stability_dash.layout.control import control_layout
+
+
+@app.callback(
+    Output("save-load-connection", "value"),
+    [Input("save-profile", "n_clicks")],
+    [State("profile-name", "value"),
+     State("runtime", "value"),
+     State("timestep", "value"),
+     State("displacement-step", "value"),
+     State("velocity-step", "value"),
+     State("acceleration-step", "value")]
+)
+def save_profile(save_clicks, profile_name, runtime, timestep, displacement, velocity, acceleration):
+    
+    if not save_clicks:
+        raise dash.exceptions.PreventUpdate
```

Comment:
This is no longer required, dash has a new feature I would like you to add to all callbacks. You may have to update your dash using pip to the latest version, and then you can add an argument within the app.callback() as follows:

```
app.callback(
prevent_initial_call = True
)
Although make sure you do it after the Out, In, and States
```